docs/pages/guides/alternatives-for-expo-notifications.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Alternatives for Expo Notifications
+sidebar_title: Expo Notifications alternatives
+description: Learn about alternative push notification services to Expo Notifications.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+
+Expo simplifies push notification implementation by abstracting much of the complexity involved in communicating with [Firebase Cloud Messaging (FCM)](https://firebase.google.com/docs/cloud-messaging) and [Apple Push Notification Service (APNs)](https://developer.apple.com/documentation/usernotifications/setting-up-a-remote-notification-server). With Expo, you can handle notifications for both Android and iOS the same way, saving development time on both the front end and back end.
+
+[Expo Notifications](/push-notifications/overview) require minimal configuration, and you can find many resources and tutorials to help you get started. Additionally, Expo offers tools like the [Expo Notifications Tool](https://expo.dev/notifications) for quick testing.
+
+For server-side notification delivery, you have multiple SDK options. The Expo team maintains the [expo-server-sdk-node](https://github.com/expo/expo-server-sdk-node), and the community provides additional SDKs ([see the full list](https://docs.expo.dev/push-notifications/sending-notifications/#third-party-sdks)). Alternatively, you can send an [HTTP/2 request](https://docs.expo.dev/push-notifications/sending-notifications/#http2-api) directly to the Expo endpoint. This API does not require authentication, making it fast and easy to use.
+
+**There is no cost** associated with sending notifications through the Expo push notification service.
+
+Using Expo Notifications is optional. You can use methods from [`expo-notifications`](/versions/latest/sdk/notifications/), such as [`getDevicePushTokenAsync`](/versions/latest/sdk/notifications/#getdevicepushtokenasync), to retrieve the native push token and use it with other services.
+
+## Limitations of Expo Notifications
+
+- **Dependency on Expo**: Your app relies on Expo's push notification service.
+- **Limited control**: Full customization, such as advanced background handling, may require direct FCM/APNs integration.
+- **Rate limits**: Expo limits push notifications to **600 notifications per second per project**. If you exceed this rate, subsequent requests will fail until the rate drops below the limit. To manage this effectively, implement request throttling (included in the [`expo-server-sdk-node`](https://github.com/expo/expo-server-sdk-node)) and add retry logic to your server.
+
+## Alternative Push Notification Providers
+
+If you need more control or additional features, consider using a third-party push notification provider. Below are some popular services that offer advanced features and integrations with Expo.
+
+### OneSignal
```

Comment:
For each of these, it would be good to offer some information that would help someone make the decision to investigate this further. Not advertising so much as it is highlighting what's important to our users. For instance, OneSignal supports AFAIK notification extensions and has client components to support that.